opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -436,6 +436,7 @@ export const stixCoreObjectImportDelete = async (context, user, fileId) => {
       prevent_indexing: true,
       context_data: contextData
     });
+    await notify(BUS_TOPICS[ABSTRACT_STIX_DOMAIN_OBJECT].EDIT_TOPIC, { ...instance, id: entityId }, user);
```

Comment:
not sure to understand why it's necessary to notify here, since it's not done for stixCoreObjectImportPush. And why ABSTRACT_STIX_DOMAIN_OBJECT ?

Reply:
The backend model has been reworked, I'm still in progress so the notify will change too.